constraint/pkg/client/client.go

```diff
@@ -715,7 +715,7 @@ func (c *Client) init() error {
 		if err != nil {
 			return fmt.Errorf("could not re-format Rego source: %v", err)
 		}
-		if err := c.backend.driver.PutModule(context.Background(), modulePath, string(src)); err != nil {
+		if err := c.backend.driver.PutModule(ctx, modulePath, string(src)); err != nil {
```

Comment:
What happens if the context is interrupted mid put-module?

Reply:
As it turns out, we don't need that Context. The issue Davis is working on will completely remove that, so I've just added a context.TODO() in `altermodules`.